### PR TITLE
docs: Correct the explanation of the node replace procedure #18978

### LINKED ISSUE #18978: docs: Issue on page Replace a Running Node in a ScyllaDB Cluster
I would like to report an issue on page https://opensource.docs.scylladb.com/branch-5.4/operating-scylla/procedures/cluster-management/replace-running-node

### Problem
The page describes the transferal of vnodes between nodes when decommissioning a node.
This is incorrect. vNodes are a number of tokens which are owned by a node, these can never be transferred between nodes. A node its tokens are permanently fixed.

The problem described occurs on multiple lines:

- "When adding a new node to a Scylla cluster, existing nodes will give the new node responsibility for a subset of their vNodes, ...:
- " When decommissioning a node from a Scylla cluster, it will give its vNodes to other nodes, making sure data is once again equally distributed."
- " Hence, replacing a node by adding and decommissioning redistribute the vNodes twice, streaming a node worth of data each time." 

To be a bit pedantic, there's no guarantee that data is equally distributed. Although, generally it is somewhat equally distributed.

###  Suggest a fix
Change the wording of this paragraph. What the author likely intends to say is that data ownership changes when a node is decommissioned. The token ranges (i.e. segments on the token ring) originally owned by the decommissioned node subsequently will become owned by other nodes. Effectively the data which maps to these token ranges will also become owned by other nodes. This data will be streamed over. 

When I talk about segments, I mean with it the following definition: "A segment is a subrange of tokens that fits entirely in one of the cluster token ranges." (source: [https://cassandra-reaper.io/docs/concepts/](https://cassandra-reaper.io/docs/concepts/)

### PR DESCRIPTION
**Pull request by vreniers**
This fixes #18978 and corrects the explanation in the docs regarding a node replace procedure.

### CODE REVIEW

File: docs/operating-scylla/procedures/cluster-management/replace-running-node.rst [hunk 1]
```diff
@@ -16,9 +16,14 @@ Add a new node to the cluster and then decommission the old node
 =================================================================
 
 Adding a new node and only then decommissioning the old node allows the cluster to keep the same level of data replication throughout the process, but at the cost of more data being transferred during the procedure.
-When adding a new node to a Scylla cluster, existing nodes will give the new node responsibility for a subset of their vNodes, making sure that data is once again equally distributed. In the process, these nodes will stream relevant data to the new node.
-When decommissioning a node from a Scylla cluster, it will give its vNodes to other nodes, making sure data is once again equally distributed. In the process, this node will stream its data to the other nodes.
-Hence, replacing a node by adding and decommissioning redistribute the vNodes twice, streaming a node worth of data each time.
+When adding a new node to a Scylla cluster, it will take up ownership of a subset of existing nodes their data.
+The new node effectively takes its place on the token ring with its own number of tokens (Vnodes), and consequentially will be responsible for data which maps to its token ranges.
```
Comment (tzach) [inline docs/operating-scylla/procedures/cluster-management/replace-running-node.rst:20]:
```suggestion
The new node effectively takes its place on the token ring with its own number of tokens (vNodes), and consequentially will be responsible for data which maps to its token ranges.
```

**Review by tzach — COMMENTED** [review]
Good work!
Two very minor comments

**Comment by vreniers** [comment]
> Good work! Two very minor comments

Thank you for review @tzach. I've adjusted it.

**Review by tzach — APPROVED** [review]

File: docs/operating-scylla/procedures/cluster-management/replace-running-node.rst [hunk 2]
```diff
@@ -16,9 +16,14 @@ Add a new node to the cluster and then decommission the old node
 =================================================================
 
 Adding a new node and only then decommissioning the old node allows the cluster to keep the same level of data replication throughout the process, but at the cost of more data being transferred during the procedure.
-When adding a new node to a Scylla cluster, existing nodes will give the new node responsibility for a subset of their vNodes, making sure that data is once again equally distributed. In the process, these nodes will stream relevant data to the new node.
-When decommissioning a node from a Scylla cluster, it will give its vNodes to other nodes, making sure data is once again equally distributed. In the process, this node will stream its data to the other nodes.
-Hence, replacing a node by adding and decommissioning redistribute the vNodes twice, streaming a node worth of data each time.
+When adding a new node to a ScyllaDB cluster, it will take up ownership of a subset of existing nodes their data.
```
Comment (nyh) [inline docs/operating-scylla/procedures/cluster-management/replace-running-node.rst:19]: The phrase "their data" doesn't seem to fit this sentence. I'm not sure what you meant to say here.

By the way, I didn't understand what wrong in the old text here? What was wrong?
By the way, both pre-patch and post-patch focus on vnodes, and it probably doesn't happen the same way with tablets.

**Comment by vreniers** [comment]
> @vreniers I'd like to have this PR merged before I add relevant sections related to tablets. Could you please:
> 
> 1. Fix the sentence on line 19 that Nadav asked about (it's "take up ownership of a subset of existing nodes their data", which should probably be "take up ownership of a subset of existing nodes and their data").
> 2. Rebase and resolve conflicts.
> 
> If you prefer, I can open a V2 of this PR myself. Please let me know.

I don't agree with that, because it can possibly imply that you are taking ownership of existing nodes (as well as their data).
We could rephrase it to: 

"When adding a new node to a ScyllaDB cluster, it may take ownership of data which belonged to other nodes."

@annastuchlik Can we agree on this?

**Comment by vreniers** [comment]
@annastuchlik Done.
